Following pxt-microbit readme throws an error at pxt serve

Trying to get a pxt dev environment set up so I can create my own target, starting by trying pxt-microbit.

Following the instructions in the [pxt-microbit README](https://github.com/Microsoft/pxt-microbit) gives the following output when running ```pxt serve```:

```
Could not determine target's pxt-core location, falling back to default: C:\Users\Adam Carriere\Documents\...\pxt
Using target PXT/microbit with build engine yotta
  Target dir:   C:\Users\Adam Carriere\Documents\...\pxt-microbit
  PXT Core dir: C:\Users\Adam Carriere\Documents\...\pxt
building node_modules/pxt-core...
[run] cd node_modules/pxt-core; jake.cmd
local pxt-core built.
building sim...
[run] cd sim; node ../node_modules/typescript/bin/tsc
Could not determine target's pxt-core location, falling back to default: C:\Users\Adam Carriere\Documents\...\pxt
building target.json in C:\Users\Adam Carriere\Documents\...\pxt-microbit...
building libs/core
  skip native build of non-project
building libs/radio
  skip native build of non-project
building libs/devices
  skip native build of non-project
building libs/bluetooth
  skip native build of non-project
building libs\blocksprj
*** yotta target bbc-microbit-classic-gcc
INTERNAL ERROR: Error: spawn yotta ENOENT
    at exports._errnoException (util.js:1018:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

Cloning the pxt-microbit repo next to the pxt repo, following the Build instructions, and running ```jake serve``` gives:

```
[serve] cd ../pxt-microbit; node ../pxt/built/pxt.js serve
Could not determine target's pxt-core location, falling back to default: C:\Users\Adam Carriere\Documents\...\pxt
Using target PXT/microbit with build engine yotta
  Target dir:   C:\Users\Adam Carriere\Documents\...\pxt-microbit
  PXT Core dir: C:\Users\Adam Carriere\Documents\...\pxt
building node_modules/pxt-core...
[run] cd node_modules/pxt-core; jake.cmd
local pxt-core built.
building sim...
[run] cd sim; node ../node_modules/typescript/bin/tsc
Could not determine target's pxt-core location, falling back to default: C:\Users\Adam Carriere\Documents\...\pxt
building target.json in C:\Users\Adam Carriere\Documents\...\pxt-microbit...
building libs/core
updating shims.d.ts (size=32980)...
  skip native build of non-project
building libs/radio
updating shims.d.ts (size=6197)...
  skip native build of non-project
building libs/devices
updating shims.d.ts (size=961)...
  skip native build of non-project
building libs/bluetooth
updating shims.d.ts (size=5607)...
  skip native build of non-project
building libs\blocksprj
*** yotta target bbc-microbit-classic-gcc
INTERNAL ERROR: Error: spawn yotta ENOENT
    at exports._errnoException (util.js:1018:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:367:16)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
jake aborted.
TypeError: task.fail is not a function
    at ChildProcess.ch.on (C:\Users\Adam Carriere\Documents\...\pxt\jakeutil.js:106:18)
    at emitTwo (events.js:106:13)
(See full trace by running task with --trace)
```

These instructions for the [Command Line Tool](https://makecode.com/cli) work fine:
```
mkdir microbit
cd microbit
pxt target microbit
pxt serve
```

It should also be noted that the [Target Creation](https://makecode.com/target-creation) instructions also fail to run with ```pxt serve```

Hopefully I'm just overlooking/missing something simple. This is all happening on Windows 10.

Almost!

Now I'm getting:
```
'srec_cat' is not recognized as an internal or external command,
operable program or batch file.
ninja: build stopped: subcommand failed.
error: command ['ninja'] failed
```
I'm assuming I need to install srec_cat now?

--- a/docs/cli.md
+++ b/docs/cli.md
@@ -10,8 +10,17 @@ npm install -g pxt
 
 ### ~ hint
 
-On Windows, make sure to install Visual Studio to get the C++ toolchains
-at https://www.visualstudio.com/downloads/
+On Windows, make sure the following are installed:
+
+1. [Yotta (follow manual install for Windows)](http://docs.yottabuild.org/#installing-on-windows)
+2. [SRecord 1.64](https://sourceforge.net/projects/srecord/files/srecord-win32/1.64/) and move it to ``` C:\```
+3. [Visual Studio and/or the C++ toolchains](https://www.visualstudio.com/downloads/)
+
+Also, make sure you add these to your Path:
+```
+C:\Python27\Scripts
+C:\srecord_dir
+```
 
 ### ~
 

